CHANGELOG.md

```diff
@@ -34,6 +38,22 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [4.0.0](https://github.com/Nike-Inc/SQift/releases/tag/4.0.0)
+
```

Comment:
Should add the release info here like we do in other releases. Should be able to go out today.

Reply:
Added in 4d9a0b3